tests/drop_counters/test_drop_counters.py

```diff
@@ -120,6 +120,82 @@ def enable_counters(duthost):
             logger.info("Restoring counter '{}' state to disable".format(port))
             duthost.command("counterpoll {} disable".format(port))
 
+
+class FanoutConfig(object):
+    def __init__(self, fanout_host, ansible_root, trunk_port, ansible_localhost):
+        self.rule_id = None
+        self.playbook = os.path.join(os.path.dirname(__file__), "exec_template.yml")
+        self.del_rule_template = os.path.join(os.path.dirname(__file__), "mlnx_fanout_del_rule.j2")
```

Comment:
hardcode for mellanox fanout

Reply:
Changed approach, now added possibility to define vendor specific module and use it inside 'fanouthost' fixture and to update/restore config.